package.json

```diff
@@ -18,7 +18,7 @@
     "tap": "7.x"
   },
   "scripts": {
-    "test": "tap --reporter tap --timeout=120 tests/api_tests.js tests/headless_test.js",
+    "test": "tap --reporter tap --timeout=120 tests/api_tests.js tests/probes/http-outbound-probe-test.js tests/headless_test.js",
```

Comment:
Could we replace this with `"tap": "tap --timeout 90 \"test/**/test-*.js\"",`? Otherwise this line is going to get a bit long. That would match any file called `test/*/test-*.js`, so you could have other helper files that just didn't start with test.